src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -207,7 +209,35 @@ def test_FV3Forecast_stage_static_files(tmp_path):
     # Ask a forecast object to stage the files to the run directory:
     forecast_obj = FV3Forecast()
     forecast_obj.create_directory_structure(run_directory)
-    forecast_obj.stage_static_files(run_directory, files_to_stage)
+    forecast_obj.stage_files(run_directory, files_to_stage)
     # Test that all of the destination files now exist:
     for dst_fn in files_to_stage.keys():
         assert (run_directory / dst_fn).is_file()
+
+
+@pytest.mark.parametrize("section", ["static", "cycledep"])
+def test_FV3Forecast_stage_files_link(tmp_path, section):
```

Comment:
I think we might be able to combine this test with the previous one by stacking parameterizations, e.g.
```
@pytest.mark.parametrize("section", ["static", "cycledep"])
@pytest.mark.parametrize("link_files", [True, False])
def test_FV3Forecast_stage_files(link_files, section, tmp_path):
...
```
(I'm not sure if the argument order matters -- I suspect not), then doing the call as
```
forecast_obj.stage_files(run_directory, files_to_stage, link_files)
```
then using the `link_files` parameter to decide whether to call `.is_file()` or `.is_symlink()`. That would only be a couple more lines of code in one method to potentially remove more duplicate code.

Reply:
Stacked parameters and now we have one test_stage_files() that tests both copy and linking functionality. Ran make format && make test. Formatter updated the format for test_forecast_run_cmd(). As I understand it, we should allow the formatter to make these changes.

Reply:
Very nice.

Yes, it's appropriate at this point to commit any changes made by `make format`. In fact, we've got a ticket in the backlog to add a CI step that tests that `make format` run in CI makes _no_ changes, meaning that we'll expect developers to have run `make format` and committed the changes themselves.

Reply:
I think the run_cmd reformatting may be an issue with your environment not reading the correct settings for the formatter.